spec/controllers/jobs_controller_spec.rb

```diff
@@ -0,0 +1,17 @@
+#encoding:utf-8
+require 'rails_helper'
+
+RSpec.describe JobsController, type: :controller do
+  let(:id) { 'afa6f8193642892fe0add387' }
+  describe 'GET status' do
+    context 'when get status with job id' do
+      it 'should return a json with status' do
```

Comment:
esse teste diz "'should return a json with status' ", mas você não verifica isso em nenhuma parte do teste.
e ja aproveitando,  mantenha seus testes sempre em um expect por teste, a não ser q seja teste de feature